vrk-kpa/suomidigi
# SUOPA-743 Twitter API PoC

To test:
- `make fresh`
- Go to https://suomidigi.docker.sh/en/admin/structure/social_media_post and click on Clear all social media... 
- When the importer has been run, go to https://suomidigi.docker.sh/en/admin/structure/social_media_post/twitter/suomidigi/list?feed_id=suomidigi&service=twitter and check that there are tweets available
